test/Microsoft.OpenApi.Tests/Validations/ValidationRuleSetTests.cs

```diff
@@ -34,7 +34,7 @@ public void DefaultRuleSetPropertyReturnsTheCorrectRules()
             // Assert
             Assert.NotNull(rules);
             Assert.NotEmpty(rules);
-            Assert.Equal(13, rules.ToList().Count); // please update the number if you add new rule.
+            Assert.Equal(14, rules.ToList().Count); // please update the number if you add new rule.
```

Comment:
I don't think this test is needed. It doesn't really add much value, and will cause maintenance issue (like the current test failure).